packages/apollo-engine-reporting-protobuf/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../tsconfig.base",
+  "compilerOptions": {
+    "rootDir": "./src",
+    "outDir": "./dist"
+  },
+  "include": ["src/**/*"],
+  "exclude": ["src/protobuf.d.ts", "src/protobuf.js"]
```

Comment:
Perhaps this is wishful thinking on my part, but this `exclude` line is now a unique exception within this entire monorepo since [with *one* other exception](https://github.com/apollographql/apollo-server/blob/f75617403fa7d7ba3c05fcd440f965c1177b3da3/packages/apollo-server-core/tsconfig.json#L8) — which we're trying to get away from in order to align them all.  I'd somewhat rather we not have `exclude` be a new type of exception.